data-pipeline/src/data_pipeline/pipelines/genes.py

```diff
@@ -199,6 +202,14 @@
     "/genes/homozygous_variant_cooccurrence_counts.ht",
 )
 
+pipeline.add_task(
+    "prepare_gnomad_v2_regional_missense_constraint",
+    prepare_gnomad_v2_regional_missense_constraint,
+    "/constraint/gnomad_v2_regional_missense_constraint.ht",
+    # TODO: before merging - update to a more permanent location for this data
```

Comment:
@rileyhgrant that reminds me, I actually did not do this for the SVs, I think if you have a current output Hail table in a dev environment, it'd be OK to take that and export it to the new ES rather than necessarily rerun the whole pipeline from scratch. So probably no need to worry about this now, we can move stuff around and fill in gaps in post.